pkg/kubelet/util/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Utility functions.
+package util
```

Comment:
`util` is a bad package name according to:
https://blog.golang.org/package-names

I know we've got util sprinkled across our code base, but is it worth avoiding introducing more? These are both related to formatting and pods, so it could be either "format" or "pod" or even methods on a *Pod or PodList ?